app/models/about.rb

```diff
@@ -1,6 +1,22 @@
 # frozen_string_literal: true
 
 class About
+  cattr_reader :plugin_stat_groups, :displayed_plugin_stat_groups
+
+  def self.add_plugin_stat_group(prefix, show_in_ui: false, &block)
+    if !@@displayed_plugin_stat_groups.include?(prefix) && show_in_ui
+      @@displayed_plugin_stat_groups << prefix
```

Comment:
I think a `Set` here will be the better data structure to use so that you don't have to worry about duplicates.